File: app/src/main/java/org/jesperancinha/xml/adder/csv/AttributeDeleteBean.java
package org.jesperancinha.xml.adder.csv;

import com.opencsv.bean.CsvBindByName;

/**
 * Created by joaofilipesabinoesperancinha on 23-02-16.
 */
public class AttributeDeleteBean {
    @CsvBindByName(column = "name")
    private String name;

    @CsvBindByName(column = "xpath")
    private String xpath;

    public String getName() {
        return name;
    }

    public String getXpath() {
        return xpath;
    }
}

File: jeorg-kotlin-apps/xml-adder/app/src/main/kotlin/org/jesperancinha/xml/adder/csv/AttributeDeleteBean.kt
package org.jesperancinha.xml.adder.csv

import com.opencsv.bean.CsvBindByName
import com.opencsv.bean.CsvBindByPosition

/**
 * Created by joaofilipesabinoesperancinha on 23-02-16.
 */
class AttributeDeleteBean {
    @CsvBindByName(column = "name")
    @CsvBindByPosition(position = 0)
    var name: String? = null

    @CsvBindByName(column = "xpath")
    @CsvBindByPosition(position = 1)
    var xpath: String? = null
}